spec/retry/exponential_backoff_strategy_spec.rb

```diff
@@ -0,0 +1,145 @@
+require 'spec_helper'
+
+RSpec.describe ActiveJob::Retry::ExponentialBackoffStrategy do
+  let(:strategy) { described_class.new(options) }
+
+  describe '#should_retry?' do
```

Comment:
similar to the `ConstantBackoffStrategy`, but I provide additional tests for `#retry_delay` below